openslides/motions/models.py

```diff
@@ -339,7 +339,11 @@ def get_new_version(self, **kwargs):
         attributes, it is populated with the data from the last version
         object if such object exists.
         """
-        new_version = MotionVersion(motion=self, **kwargs)
+        if self.pk is None:
+            # Do not reference the MotionVersion object to an unsaved motion
```

Comment:
Ok but then what happens if I try to save such a motion. Do we need a fix in save()?